### PR TITLE
Support copyFiles regardless of runtime

### LINKED ISSUE #635: Support copyFiles regardless of runtime
CopyFiles is a useful feature regardless of the runtime so we should pull this out of the runtime specific props.

### PR DESCRIPTION
**Pull request by thdxr**
Fixes #635 

I started to write a test but wasn't sure if there was a recommended way to check what's in `./build` for the specific handler

### CODE REVIEW

File: packages/resources/src/Function.ts [hunk 1]
```diff
@@ -320,15 +328,37 @@ export class Function extends lambda.Function {
...
-    return layers.map(layer => {
+  static copyFiles(
+    bundle: FunctionBundleProp | undefined,
+    srcPath: string,
+    buildPath: string
+  ) {
+    if (!bundle) return;
+    if (typeof bundle === "boolean") return;
+    if (!bundle.copyFiles) return;
+
```
Comment (thdxr) [inline packages/resources/src/Function.ts:339]: This is using Typescript type narrowing to allow it to infer the right type

File: packages/resources/src/Function.ts [hunk 2]
```diff
@@ -320,15 +328,37 @@ export class Function extends lambda.Function {
...
+    bundle.copyFiles.forEach(({ from, to }) => {
+      const fromPath = path.join(srcPath, from);
+      const toPath = path.join(buildPath, to);
+      console.log(process.cwd(), fromPath, toPath);
```
Comment (fwang) [inline packages/resources/src/Function.ts:343]: Debugging log?

Comment (thdxr) [reply packages/resources/src/Function.ts:343]: Good catch :sweat_smile:

**Comment by fwang** [comment]
> I started to write a test but wasn't sure if there was a recommended way to check what's in `./build` for the specific handler

We don't have a standard way of checking it. We have an existing test for `copyFiles` that just calls `fs.existsSync` to test for specific files.
https://github.com/serverless-stack/serverless-stack/blob/master/packages/cli/test/nodejs-build-bundle-copyfiles/nodejs-build-bundle-copyfiles.test.js

**Review by fwang — APPROVED** [review]

**Pull request merged**